insights/parsers/smartctl.py

```diff
@@ -202,3 +211,53 @@ def parse_attributes(line):
 
         # Delete temporary full line storage
         del self.full_line
+
+
+@parser(Specs.smartctl_l_scterc)
+class SMARTctlSCTERC(CommandParser):
+    """
+    Parser for output of ``smartctl -l scterc`` for each drive in system.
+
+    This stores the SCT ERC (Smart Command Transfer Error Recovery Control) information
+    from the output of `smartctl -l scterc` in the following properties:
+    following properties:
+
+    * ``device`` - the name of the device after /dev/ - e.g. sda
+    * ``scterc`` - the SCT Error Recovery Control settings - e.g. 'Read': 20 seconds
+
+    Sample output::
+
+        smartctl 7.1 2020-04-05 r5049 [x86_64-linux-4.18.0-240.el8.x86_64] (local build)
+        Copyright (C) 2002-19, Bruce Allen, Christian Franke, www.smartmontools.org
+        SCT Error Recovery Control set to:
+         Read: 200 (20.0 seconds)
+         Write: 200 (20.0 seconds)
+
+    Examples:
+        >>> smartctl.device
+        '/dev/sda'
+        >>> smartctl.scterc['Read']
+        20.0
+        >>> smartctl.scterc['Write']
+        20.0
+
+    """
+
+    def __init__(self, context):
+        filename_re = re.compile(r'smartctl_-l_scterc_\.dev\.(?P<device>\w+)$')
+        match = filename_re.search(context.path)
+        if match:
+            self.device = '/dev/' + match.group('device')
+        else:
+            raise ParseException('Cannot parse device name from path {p}'.format(p=context.path))
+        super(SMARTctlSCTERC, self).__init__(context)
+
+    def parse_content(self, content):
+        self.scterc = {}
```

Comment:
@shlao since this parser is primarily just a `dict` for the parsed data why not use `dict` for the base class.  See this [example parser](https://github.com/RedHatInsights/insights-core/blob/master/insights/parsers/rndc_status.py).